app/src/main/java/com/kickstarter/services/apiresponses/commentresponse/PageInfoEnvelope.kt

```diff
@@ -0,0 +1,29 @@
+package com.kickstarter.services.apiresponses.commentresponse
+
+class PageInfoEnvelope private constructor(
```

Comment:
I do love that you tried to use the data classes here, but there is still missing the annotations for making this envelope able to be send as an intent object, how are those annotations gonna behave in kotlin data classes?
`@AutoGson
@AutoParcel`